.github/workflows/macos.yml

```diff
@@ -0,0 +1,21 @@
+name: macos
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
```

Comment:
FWIW I'm also doing a `paths-ignore` in the Linux jobs but I'm not sure it's working 100%. I'm trying to avoid having the build run for documentation changes, or non-C code changes...

Reply:
Although I'm not confident that the current paths-ignore configuration actually has the intended effect. I'm still seeing jobs kicked off when NEWS and README.me are changed.